app/classes/document_search_results.rb

```diff
@@ -1,13 +1,14 @@
 # frozen_string_literal: true
 
 class DocumentSearchResults
-  attr_reader :total, :offset, :results, :suggestion
+  attr_reader :total, :offset, :results, :suggestion, :aggregations
 
   def initialize(result, offset = 0)
     @total = result['hits']['total']
     @offset = offset
     @results = extract_hits(result['hits']['hits'])
     @suggestion = extract_suggestion(result['suggest'])
+    @aggregations = extract_aggregations(result['aggregations'])
```

Comment:
These file changes look good. I realized we have no unit tests for this class; we should probably add those at some point while we are doing more I14y work.

Reply:
Took a shot at a rudimentary spec now [here](https://github.com/GSA/i14y/blob/1abb8d62c3f024316e466ba8e55c310975b75e9b/spec/classes/document_search_results_spec.rb).